### PR TITLE
Fix admin-dl-horizontal

### PR DESCRIPTION
**Pull request by wxiaoguang**
image

image

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Comment by silverwind** [comment]
Seems better with no wrapping on desktop, yes. I guess it may still wrap on some locales.

File: web_src/css/admin.css [hunk 1]
```diff
@@ -15,32 +15,27 @@
...
-
-/* divider needs explicit width to become visible here */
+/* divider needs explicit width because the parent container is "display: flex" */
 .admin dl.admin-dl-horizontal .ui.divider {
   width: 100%;
 }
 
+.admin dl.admin-dl-horizontal dt,
+.admin dl.admin-dl-horizontal dd {
+  line-height: 1em;
```
Comment (silverwind) [inline web_src/css/admin.css]:
```suggestion
  line-height: 1;
```

From [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height):

> The used value is this unitless [<number>](https://developer.mozilla.org/en-US/docs/Web/CSS/number) multiplied by the element's own font size. The computed value is the same as the specified <number>. In most cases, this is the preferred way to set line-height and avoid unexpected results due to inheritance.

**Review by silverwind — APPROVED** [review]

**Comment by silverwind** [comment]
No additional changes necessary now that https://github.com/go-gitea/gitea/pull/25516 has merged, right?

**Comment by wxiaoguang** [comment]
I think it's ready to merge.

**Pull request merged**